Userland/Libraries/LibCore/StandardPaths.cpp

```diff
@@ -62,6 +62,8 @@ DeprecatedString StandardPaths::config_directory()
     builder.append(home_directory());
 #if defined(AK_OS_MACOS)
     builder.append("/Library/Preferences"sv);
+#elif defined(AK_OS_HAIKU)
+    builder.append("/config/settings"sv);
```

Comment:
I think this and other additions to this file shouldn't be needed, since the `XDG` environs are declared on Haiku?

Reply:
XDG_CONFIG_HOME and XDG_DATA_HOME are declared on Haiku, but XDG_RUNTIME_DIR isn't and that makes Ladybird crash here, and our font directory differs from those declared in #else.
Therefore modifications to this file are needed.
I decided to properly declare the config_directory and data_directory to Haikus default directories as well for the sake of completeness and consistency.

Reply:
For correctness, this should really use Haiku's `find_paths` API, as there's multiple fonts directories and the like.

Reply:
Thanks for the hint to the find_paths() function.
I changed my code to use it for getting the font directories.
For the other directories, we only need one single directory where we put our stuff and I think it's fine to hardcode those  to Haikus defaults instead of using find_directory().